app/build.gradle

```diff
@@ -41,6 +41,17 @@ dependencies {
     implementation 'com.google.firebase:firebase-ml-vision:24.0.0'
     // Face model
     implementation 'com.google.firebase:firebase-ml-vision-face-model:19.0.0'
+    implementation 'com.google.firebase:firebase-analytics:17.2.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'com.android.support:design:28.0.0'
+    implementation 'com.android.support:support-v4:28.0.0'
+    implementation 'com.android.support:support-vector-drawable:28.0.0'
```

Comment:
Use the androidx namespace for consistency

Reply:
androidx dile gradle build e problem hocche

Reply:
Please share the error message